tests/Language/Haskell/Stylish/Step/Data/Tests.hs

```diff
@@ -298,71 +319,73 @@ case15 = expected @=? testStep (step 2) input
     expected = unlines
       [ "module Herp where"
       , ""
-      , "data Foo = Foo {"
-      , "   a :: Int -- ^ comment"
+      , "data Foo = Foo"
+      , "  { a :: Int -- ^ comment"
       , "  }"
       ]
 
-case16 :: Assertion
-case16 = expected @=? testStep (step 2) input
+case17 :: Assertion
+case17 = expected @=? testStep (step 2) input
   where
     input = unlines
        [ "module Herp where"
        , ""
        , "data Foo a = Foo"
        , "  { a :: a,"
-       , "-- ^ comment"
+       , "-- comment"
        , "   a2 :: String"
        , "  }"
        ]
     expected = unlines
        [ "module Herp where"
        , ""
        , "data Foo a = Foo"
-       , "  { a :: a,"
-       , "-- ^ comment"
-       , "   a2 :: String"
+       , "  { a :: a"
+       , "   -- comment"
```

Comment:
This comment is a bit off to the right, can you move it 1 space to the left?

Reply:
👍